commercial-paper/organization/digibank/application-go/business/addtowallet.go

```diff
@@ -0,0 +1,95 @@
+package business
+
+import (
+	"io/ioutil"
```

Comment:
I'd suggest avoiding **ioutil** as it has been deprecated since Go 1.16. All the functionality in **ioutil** is available in either the **io** or **os** packages

Reply:
Thank you very much for your reply, this is indeed a problem that deserves attention!